### ISSUE TITLE
[IMPORTANT] Gitlab migration

### ISSUE THREAD

**Issue by lbartoletti** [Oslandia/SFCGAL#230]
Oslandia migrates from GitHub to GitLab

Some of you may have noticed, we recently migrated all our public projects from GitHub to GitLab.

On Monday June 29th, we will migrate this project to gitlab (https://gitlab.com/oslandia/sfcgal).

### Practical aspects of the migration

The migration had some consequences, and if you have been involved in some of our public projects, you may have noticed some changes. Please note the following points.

- If you have no GitLab account yet, then you can directly login to GitLab with your GitHub account
- When we did the migration, some issues where changed owners if their original author had no GitLab.com account at the time. This is mentioned in the new issue as a note
- Travis CI is not supported by GitLab, and we will port travis CI to GitLab CI for projects needing it
- Git remote paths have changed and should be updated wherever needed
- All our projects on GitHub are now archived and read-only, with mentions of the new home at GitLab.com

There are probably implications that we did not foresee. Should you be impacted by this migration, please contact us and we will try to fix the issue as soon as possible.

@1SpatialGroupLtd @admin-zoujing @Algunenano @amutu @christianurich @coolbole @cuulee @danfran @danielcu888 @danielcumberbatch @DRC1Spatial @elmalba @ezhangle @Ezio47 @fxcoudert @geoware @giggls @giohappy @giserh @happydpc @hdeeken @HEShuang @hgryoo @hjanetzek @iut-ibk @jiapei100 @jjcorreao @jjpepper @kashif @Komzpa @konturio @lbartoletti @lidi100 @matze09 @mborne @metux @mhugo @mthh @oss-qm @robe2 @roman0yurin @sanak @sebastic @shan23 @sloriot @smathermather @strk @stromnov @StyXman @thomersch @tibnor @tigerfoot @tinlyx @troopa81 @vitaliy-gis @vivzqs @WillieMaddox @xdlwd086 @zhatin @zjanetzek @SebastienPeillet @delhomer

**Comment by sebastic** [comment]
What about www.sfcgal.org, that still seems to use oslandia.github.io?

**Comment by sebastic** [comment]
> All our projects on GitHub are now archived and read-only, with mentions of the new home at GitLab.com

This seems to be the other way around, the project on GitLab shows:
>  /!\ Deletion pending. This project will be removed on 2020-06-23. Repository and other project resources are read-only.

**Comment by thomersch** [comment]
You probably have very good reasons for switching, but you externalize some work to others: Now I gotta create a gitlab account, resubscribe to the issues I am interested in and in addition some comments from some issues are now gone in gitlab, for no apparent benefit.

**Comment by strk** [comment]
> Now I gotta create a gitlab account

You don't, can use your github account to login

> resubscribe to the issues I am interested in

You probably do. In some extent is a good way to check who's still
interested in those issues....

> some comments from some issues are now gone in gitlab, for no apparent benefit.

Sounds like a bug, can you be more specific ?

To me the benefit is very much apparent: avoid lock-in in a
proprietary hosting platform

**Comment by thomersch** [comment]
> You don't, can use your github account to login

Which gives me a Gitlab account. It doesn‘t really change anything except that I don‘t have to save an additional password. People already have more than enough accounts.

> You probably do. In some extent is a good way to check who's still interested in those issues....

To be blunt, I find it a little bit condescending that you assume what is better for everyone anyway.

> To me the benefit is very much apparent: avoid lock-in in a proprietary hosting platform

gitlab.com is proprietary. They have a separate open source application, but that‘s something different.

> Sounds like a bug, can you be more specific ?

Stuff here is missing: https://gitlab.com/Oslandia/SFCGAL/-/issues/202

Honestly, I don‘t care so much about all of this, but people are busy making stuff and we don‘t need additional disruptions, that‘s all I wanted to get across.

**Comment by lbartoletti** [comment]
> > 
> 
> This seems to be the other way around, the project on GitLab shows:
> 
> > /!\ Deletion pending. This project will be removed on 2020-06-23. Repository and other project resources are read-only.

@sebastic Yes this project will be removed (it was a test for our team) and recreated the 29th June

**Comment by lbartoletti** [comment]
> What about [www.sfcgal.org](http://www.sfcgal.org), that still seems to use oslandia.github.io?

And it's not updated since years. We're working on it.

**Comment by lbartoletti** [comment]
For your information : https://oslandia.com/en/2020/06/19/en-oslandia-migrates-from-github-to-gitlab/
